test/cmd/generic-resources.sh

```diff
@@ -445,7 +445,7 @@ run_recursive_resources_tests() {
   # Post-condition: busybox0 & busybox1 should error as they are RC's, and since busybox2 is malformed, it should error
```

Comment:
This comment suggests `busybox2` also has an error... shoud we be checking for that as well?

Reply:
no, the actual manifest is malformed, no API request is submitted